# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-rustfmt.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-144.0.7559.59/build/rust/rust_bindgen_generator.gni.vanilla	2026-01-14 17:58:30.652002111 +0100
+++ chromium-144.0.7559.59/build/rust/rust_bindgen_generator.gni	2026-01-14 17:59:05.743453044 +0100
@@ -16,7 +16,7 @@
   _bindgen_path = "${_bindgen_path}.exe"
 }
 
-_rustfmt_path = "${rust_bindgen_root}/bin/rustfmt"
+_rustfmt_path = "/bin/true"
 if (!use_chromium_rust_toolchain &&
     (host_cpu == "ppc64" || host_cpu == "s390x")) {
   _rustfmt_path = rust_sysroot_absolute + "/bin/rustfmt"
